### PR TITLE
Update ACCEPT/DISPLAY for MF

### PR DESCRIPTION
**Pull request by ddeclerck**

### CODE REVIEW

**Review by nberth — APPROVED** [review]
All fine by me!

File: src/lsp/cobol_parser/grammar_tokens.mly [hunk 1]
```diff
@@ -138,8 +138,8 @@
 %token BITMAP_TRAILING              [@keyword]                [@contexts ]
 %token BITMAP_TRANSPARENT_COLOR     [@keyword]                [@contexts ]
 %token BITMAP_WIDTH                 [@keyword]                [@contexts ]
-%token BLANK                        [@keyword]
-%token BLINK                        [@keyword]                [@contexts screen_descr_entry, set_attribute_stmt]
+%token BLANK                        [@keyword]                [@contexts display_stmt, accept_stmt]
```
Comment (nberth) [inline src/lsp/cobol_parser/grammar_tokens.mly:141]: I think that one should not become context-sensitive.

**Review by nberth — CHANGES_REQUESTED** [review]

File: src/lsp/cobol_parser/grammar_tokens.mly (hunk 1 above)
Comment (ddeclerck) [reply src/lsp/cobol_parser/grammar_tokens.mly]: I removed the context on `BLANK`.

**Review by nberth — APPROVED** [review]

**Comment by nberth** [comment]
Note: we seem to loose some performances with the new context for `DISPLAY`. This needs further investigations, but that PR can be merged in the meantime.

**Pull request merged**